### ISSUE TITLE
Build Get Help

### ISSUE THREAD

**Issue by candilemoine** [department-of-veterans-affairs/va.gov-team#103715]
#### Mockups/Designs

- [ Component ](link)

### Acceptance Criteria/Conditions of Satisfaction

- [ ] 

#### Accessibility (A11y) Testing 
- Refer to existing [VA design system](https://design.va.gov/components/form/)
- Test with either [WAVE](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US&utm_source=ext_sidebar) or [AXE](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US&utm_source=ext_sidebar) extension
- As you manually test, look for the following:
   - color contrast errors
   - keyboard navigation
     - When navigating with a keyboard, content should flow from left to right
   - heading order 
     - Headings should be in order, so h1, then h2, h3. This is so the screen reader can let the user know the natural order of the elements in page. The WAVE tool has a headings tab that will display the order.
   - proper tag elements
     - Since we are working with forms, make sure components have the proper tags, i.e.:
 <label for="lastName"...>Last Name</label> <input id="lastName"... />
- test with a screen reader 
  - Voiceover is built in with mac, use NVDA for windows

**Comment by jquispe-oddball** [comment]
@seipeljl added breadcrumb as part of a/c

**Comment by jquispe-oddball** [comment]
@candilemoine adding pdf file inside vets-website assets folder as discussed in 4/3 stand up. I think we should still try to figure out what is the standard the VA uses to host files.

**Comment by candilemoine** [comment]
@jquispe-oddball Platform uses AWS S3 bucket. Been working with the Platform team to give us access. [Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/103857)

**Comment by jquispe-oddball** [comment]
@candilemoine thanks! the get help page is on staging, but the download link doesn't work. how long do you think it will take to host the file in aws?

<img width="1121" alt="Image" src="https://github.com/user-attachments/assets/7151fc00-9739-42af-af2c-6ad1ab778822" />

**Comment by jquispe-oddball** [comment]
@candilemoine forgot to mention, once this is good to go we need another ticket to add the link back to the desktop/mobile nav

**Comment by seipeljl** [comment]
@candilemoine @jquispe-oddball page styling looks good!  

Two behaviors I noticed that don't match the AC:
- the email does not have a pre-populated subject line
- the link to VA Form 21-22 is not opening in a new tab

As far as VA Form 21-22 not opening in a new tab... I think we could remove that AC for this page. We received feedback during Midpoint that we only needed "open in new tab" behavior when the user would lose data if they were directed away from the current page, which isn't happening here.

**Comment by candilemoine** [comment]
@jquispe-oddball @seipeljl created the [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/107396) to add it back. We'll add it later. Waiting to hear back from Platform on the S3 bucket.

**Comment by candilemoine** [comment]
Here's the [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/107425) to track S3 bucket access progress

I posted in the other ticket as well.

**Comment by jquispe-oddball** [comment]
@candilemoine can we move this to blocked until we have the right download link? Oren gave me some direction on how to possibly[ fix the download](https://dsva.slack.com/archives/C06ABHUNBRS/p1744219294690539?thread_ts=1743778877.579539&cid=C06ABHUNBRS), and it's still not working. Right now the download link just goes directly to github.
